src/ui_utils.c

```diff
@@ -2566,6 +2582,10 @@ void ui_init(void)
 	ui_init_toolbar_widgets();
 	init_document_widgets();
 	create_config_files_menu();
+
+	// after UI is initialized, apply user's custom CSS (if it exists)
+	// to override other CSS styles
+	init_user_style();
```

Comment:
@b4n ah, right, it was this:

```
Geany-INFO: Using alternate configuration directory
Geany-INFO: Geany 1.30 (git >= 30a8921), en_CA.UTF-8
Geany-INFO: GTK 3.18.9, GLib 2.48.2
Geany-INFO: System data dir: /opt/geany/share/geany
Geany-INFO: User config dir: /tmp/gtkcsstestconfig
Geany-INFO: Loaded GTK+ CSS theme '/opt/geany/share/geany/geany.css'

(geany:27568): Gtk-CRITICAL **: gtk_container_add: assertion 'GTK_IS_CONTAINER (container)' failed
Geany-INFO: System plugin path: /opt/geany/lib/geany
Geany-INFO: Added filetype JSON (61).
Geany-INFO: Added filetype Cython (62).
Geany-INFO: Added filetype Scala (63).
Geany-INFO: Added filetype CUDA (64).
Geany-INFO: Added filetype Genie (65).
Geany-INFO: Added filetype Clojure (66).
Geany-INFO: Added filetype Graphviz (67).
Geany-INFO: Added filetype Arduino (68).
Geany-INFO: Loaded libvte from libvte-2.91.so
Geany-INFO: unknown : None (UTF-8)
```

I could figure it out, but doing it after fixes this.